integration-test/tests/graph_editor.rs

```diff
@@ -0,0 +1,31 @@
+use enso_frp::future::EventOutputExt;
+use enso_integration_test::IntegrationTest;
+use wasm_bindgen_test::wasm_bindgen_test;
+
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
+async fn create_new_project_and_add_nodes() {
+    let test = IntegrationTest::setup().await;
+    let ide = &test.ide;
+    let project = ide.presenter.view().project();
+    let graph_editor = project.graph();
+    let controller = ide.presenter.controller();
+    let project_management =
+        controller.manage_projects().expect("Should be able to manage projects");
```

Comment:
What is meant with `"Should be able to manage projects"`? Is this supposed to be an explanation of the guarantee that this never fails, or an error message? In either case, I'm uncertain whether this conveys what is going on.